specification/purview/data-plane/Azure.Analytics.Purview.MetadataPolicies/preview/2020-12-01-preview/purviewmetadatapolicies.json

```diff
@@ -0,0 +1,648 @@
+{
```

Comment:
Examples are required for all operations.  Take a look at [this](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2021-05-01-preview/purviewcatalog.json#L53) for an example of how these are wired in.

Reply:
Added examples for all operations. Thanks.

Reply:
Looks like some error in examples, will check.

Reply:
> Examples are required for all operations. Take a look at [this](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2021-05-01-preview/purviewcatalog.json#L53) for an example of how these are wired in.

All validations are passing now.